instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/SpanKey.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.instrumenter;
+
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.ContextKey;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+// TODO make class (and fields/methods below) package-private after tracers are gone and no need for
+//  bridging from ClientSpan/ServerSpan/ConsumerSpan
```

Comment:
it seems @mateuszrzeszutek has a use-case for them bring public so they can be used in native lib/manual instrumentations